latch/types/directory.py

```diff
@@ -102,9 +102,14 @@ def local_path(self) -> str:
 
     @property
     def remote_path(self) -> Optional[str]:
-        """A url referencing in object in LatchData or s3."""
+        """A url referencing a path prefix in LatchData or s3."""
         return self._remote_directory
-
+    
+    @property
+    def maybe_remote_path(self) -> str:
```

Comment:
this name might be confusing since "maybe" implies an optional return.  How about `resolved_path`?